collab-database/src/rows/row.rs

```diff
@@ -213,24 +213,30 @@ impl BorrowMut<Collab> for DatabaseRow {
 
 pub struct DatabaseRowBody {
   row_id: RowId,
-  data: MapRef,
+  pub data: MapRef,
   #[allow(dead_code)]
-  meta: MapRef,
+  pub meta: MapRef,
   #[allow(dead_code)]
   comments: ArrayRef,
 }
 
 impl DatabaseRowBody {
-  pub fn open(row_id: RowId, collab: &mut Collab) -> Result<Self, DatabaseError> {
+  pub fn open(collab: &mut Collab) -> Result<Self, DatabaseError> {
     CollabType::DatabaseRow.validate_require_data(collab)?;
-    Ok(Self::create_with_data(row_id, collab, None))
+    Self::create_with_data(collab, None)
   }
 
-  pub fn create(row_id: RowId, collab: &mut Collab, row: Row) -> Self {
-    Self::create_with_data(row_id, collab, Some(row))
+  pub fn create(collab: &mut Collab, row: Row) -> Result<Self, DatabaseError> {
+    Self::create_with_data(collab, Some(row))
   }
 
-  fn create_with_data(row_id: RowId, collab: &mut Collab, row: Option<Row>) -> Self {
+  fn create_with_data(collab: &mut Collab, row: Option<Row>) -> Result<Self, DatabaseError> {
+    let row_id: String = collab
```

Comment:
when create a row, the database row_id must be empty

Reply:
reverted